docs/REFERENCE.md

```diff
@@ -2606,6 +2607,12 @@ used.
 
 ---
 
+### `stop`
+
+`stop` stops template execution, without returning an error.
```

Comment:
Ops, in fact I did need to error (making the chezmoi call return a non-zero code). Perhaps this function can accept a parameter like `exitCode`, which defaults to 0 and can be changed on a need basis?

Reply:
OK, PR updated. PTAL.